test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCreateSuite.java

```diff
@@ -160,8 +160,8 @@ public List<HapiApiSpec> getSpecsInSuite() {
 						contractWithAutoRenewNeedSignatures(),
 						autoAssociationSlotsAppearsInInfo(),
 						getsInsufficientPayerBalanceIfSendingAccountCanPayEverythingButServiceFee(),
-						canCallPendingContractSafely(),
 						createContractWithStakingFields()
+//						canCallPendingContractSafely(),
```

Comment:
Should this be uncommented?

Reply:
This one is always flaky when run in the GitHub actions workflow. 🤔